models/edr/alerts/alerts_anomaly_detection.sql

```diff
@@ -15,7 +15,7 @@ alerts_anomaly_detection as (
            test_execution_id,
            test_unique_id,
            model_unique_id,
-           detected_at,
+           {{ elementary.edr_cast_as_timestamp("detected_at") }} as detected_at,
```

Comment:
Originally it was because Athena had issues handling timestamp types in views with iceberg tables. But it works without problems now. Looks like Athena fixed the issue in July. I will remove those and re-test. Great you asked! :)
...........
from https://docs.aws.amazon.com/athena/latest/ug/release-notes.html

July 24, 2023
Published on 2023-07-24
Athena announces the following fixes and improvements.
....
Iceberg views – Fixed a compatibility issue with the precision of timestamp columns in Apache Iceberg views. Iceberg views that have timestamp columns are now readable regardless of whether the columns were created on Athena engine version 2 or Athena engine version 3.

Reply:
fixed